packages/react-instantsearch/examples/autocomplete/package.json

```diff
@@ -23,7 +24,9 @@
     "notify": false,
     "moduleNameMapper": {
       "\\.(css)$": "identity-obj-proxy"
-    }
+    },
+    "setupFiles": ["../../../../shim.js"],
```

Comment:
Maybe you can use <ROOT_DIR> smg here?

Reply:
Could be better indeed :)

Reply:
@vvo is <rootDir> here referring to the directory where the **example** package.json is, and therefore, can't work if shim.js is only at the very root of the project?

Reply:
Indeed root_dir would be the example/package.json.

Also this seems temporary given comments in https://github.com/facebookincubator/create-react-app/issues/3199, just watch the issue and remove it afterwards